examples/tutorial.ipynb

```diff
@@ -166,11 +170,84 @@
     "for r in results:\n",
     "    rasterio.plot.show(rasterio.open(r.result()))"
    ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "45a14473",
+   "metadata": {},
+   "source": [
+    "We can also specify a grid using the UMM grid name from the CMR. Let's build another request. This time, we'll request a specific granule and grid."
```

Comment:
How does a user find what gird names are available? Could we include a CMR query that provides the associated grids available for that collection? And information on what those grid parameters are?

Reply:
Sent a reply in Slack.

Reply:
<img width="1196" alt="Screen Shot 2023-02-24 at 2 59 44 PM" src="https://user-images.githubusercontent.com/34752045/221278988-c7391e54-5910-4221-a910-047f8e1beeb3.png">